### PR TITLE
Fortran for macOS 3 samples DRAFT

### PR DESCRIPTION
**Pull request by greenrongreen**
Initial DRAFT Fortran macOS samples (3).  vec_samples  optimize_samples. openmp_samples

### CODE REVIEW

**Comment by anjgola** [comment]
Can you please un-draft this so that CI can run.

File: DirectProgramming/Fortran/openmp_samples/sample.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "openmp_samples",
+  "categories": [ "Toolkit/Intel® oneAPI HPC Toolkit" ],
+  "description": "Fortran Tutorial - Using OpenMP",
+  "toolchain": [ "ifort" ],
+  "languages": [ { "fortran": {} } ],
+  "targetDevice": [ "CPU" ],
+  "os": [ "darwin" ],
+  "builder": [ "make" ]
+}
```
Comment (mkitez) [inline DirectProgramming/Fortran/openmp_samples/sample.json:10]: Please, add the `"ciTests"` field according to https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object

**Comment by greenrongreen** [comment]
I don’t see how to un-draft this – is it the
“Publish your first package” link?

[cid:image001.png@01D656B6.CEAA1D20]

**Comment by greenrongreen** [comment]
These 3 samples were ported over from our Parallel Studio XE samples, for macOS.
These work with the early build of the oneAPI Base + HPC kit for macOS.  
As an FYI they would work with Linux also but I was only asked for macOS.

Do we need to ask Paul Petersen to review also??

**Comment by JoeOster** [comment]
Paul Petereson should review as well

File: DirectProgramming/Fortran/openmp_samples/sample.json (hunk 1 above)
Comment (anjgola) [reply DirectProgramming/Fortran/openmp_samples/sample.json:10]: @greenrongreen  This part is what is preventing this merge mostly. Please add the ciTests field as mentioned by Roman.

**Comment by greenrongreen** [comment]
3 json files were edited to include CI stanzas.  those 3 files pushed today.

File: DirectProgramming/Fortran/openmp_samples/sample.json (hunk 1 above)
Comment (greenrongreen) [reply DirectProgramming/Fortran/openmp_samples/sample.json:30]: added the CI stanzas and pushed the 3 json files

**Comment by greenrongreen** [comment]
@JoeOster I have pushed the revised 3 README.md files that match your example's outline

**Comment by greenrongreen** [comment]
ah, I did not know about jsonlint.  I have a fix to push

**Pull request merged**